### DISCUSSION TITLE
can this be configured to pay from a separate wallet?

### DISCUSSION THREAD

**Discussion by mboyd1** [coinfoundry/miningcore#1352] [Q&A]
Instead of paying out from the wallet that receives the freshly mined coinbase rewards, can it be configured so that the coinbase address is an offline cold wallet, and the payouts are sent from a different wallet that is kept topped-up manually by the pool operator?

In this scenario, the freshly mined coins would stack up in cold storage wallet 'A', while the pool pays out daily from a hot wallet 'B' that is a different address.  The calculated amounts paid to each hasher remains the same it just comes from another wallet.

**Comment by oliverw** [comment]
No sorry.
